lib/categories/words/words1@sayutizxc/Words1HomePage.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:riddleworld/universal/result.dart';
+import 'package:riddleworld/universal/riddleAppbar.dart';
+
+class Words1HomePage extends StatefulWidget {
+  @override
+  _Words1HomePageState createState() => _Words1HomePageState();
+}
+
+class _Words1HomePageState extends State<Words1HomePage> {
+  final TextEditingController _controller = TextEditingController();
+  bool _answer = false;
+  int _score = 0;
+  int _count = 0;
+  void resetHanlder() {
+    setState(() {
+      _answer = false;
+      _count = 0;
+      _score = 0;
+      _controller.text = '';
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return RiddleBar(
+      title: 'FIND THE HIDDEN WORDS',
+      body: (_answer)
+          ? Result(_score, 1, resetHanlder, '/wordPage')
+          : Center(
+              child: ListView(
+                children: <Widget>[
+                  SizedBox(
+                    height: 10,
+                  ),
+                  (TimeOfDay.now() != TimeOfDay(hour: 00, minute: 00))
+                      ? IconButton(
+                          onPressed: () {
+                            setState(() {
+                              _count++;
+                            });
+                          },
+                          icon: Icon(Icons.brightness_3))
+                      : Center(child: Text('FROM THE DARK')),
+                  Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Container(
+                        padding: EdgeInsets.all(10),
+                        decoration: BoxDecoration(
+                            border: Border.all(width: 1.6),
+                            color: Colors.transparent),
+                        width: MediaQuery.of(context).size.width * 0.9,
+                        child: Text(
+                          (_count > 10) ? '00' : 'TAP THE MOON',
```

Comment:
If we use the 00:00 instead of just 00, it would be easy for players to guess! can you change it and give us PR once again!!?

Reply:
yes that's a good idea, i will change it